invokeai/app/api/routers/images.py

```diff
@@ -84,6 +84,17 @@ async def delete_image(
         # TODO: Does this need any exception handling at all?
         pass
 
+@images_router.post("/clear-intermediates", operation_id="clear_intermediates")
```

Comment:
I think there should be two routes:
- `GET /intermediates` returns the total number of intermediates
- `POST /intermediates` with `count: int = Body(...)` deletes that many intermediates

These would be a simple query/mutation pair, where the query tags are invalidated by the mutation. Nothing else involved in the RTK Query side.

Then we could just show exactly how many intermediates there are to clear.

The UI would still only do 100 at a time (I think that's reasonable).

Reply:
Another option is to return `count_deleted` and `count_remaining`, but I think it's better to have two routes.